lib/logstash/outputs/CiscoZeus.rb

```diff
@@ -22,9 +22,11 @@ def register
   end # def register
 
   def multi_receive(events)
-    result = @zeus_client.send_logs(@log_name, events)
-    if not result.success?
-      STDERR.puts "Failed to send data to zeus: " + result.data.to_s
+    events.group_by{ |ev| ev.sprintf(@log_name) }.each do |log_name, events_group| 
+      result = @zeus_client.send_logs(log_name, events_group)
+      if not result.success?
```

Comment:
ohh ruby is beautiful, but I think in this case `if not` is more readable. "If the result is not successful print to stderr" sounds more logical than "Unless the result is successful print to stderr"